tapdb/migrations_test.go

```diff
@@ -130,3 +132,64 @@ func TestMigrationDowngrade(t *testing.T) {
 	err := db.ExecuteMigrations(TargetLatest, WithLatestVersion(1))
 	require.ErrorIs(t, err, ErrMigrationDowngrade)
 }
+
+func TestMigration20(t *testing.T) {
+	ctx := context.Background()
+
+	db := NewTestDBWithVersion(t, 19)
+
+	// We need to insert some test data that will be affected by the
+	// migration number 20.
+	InsertTestdata(t, db.BaseDB, "migrations_test_00020_dummy_data.sql")
+
+	// And now that we have test data inserted, we can migrate to the latest
+	// version.
+	err := db.ExecuteMigrations(TargetLatest)
+	require.NoError(t, err)
+
+	_, assetStore := newAssetStoreFromDB(db.BaseDB)
```

Comment:
At this point in the test, we are primarily verifying the state of the database after it has been loaded. We should add a comment to clarify this.